src/config/config.service.ts

```diff
@@ -24,6 +25,11 @@ export class ConfigService {
         this.globalConfig = this.loadGlobalConfig();
         this.chainsConfig = this.loadChainsConfig();
         this.ambsConfig = this.loadAMBsConfig();
+        this.initialize();
```

Comment:
Save the promise to a `this.isReady` `readonly` variable and await that promise before the app initialization (i.e. before the line `await app.listen(...)` on `main.ts`)

Reply:
Check if I got the right idea on this comment

Reply:
You are missing awaiting the `isReady` promise on `main.ts`

Reply:
Check if now is what you are saying. I think i got it but I'm no 100% sure